### PR TITLE
Rename components

### PR DESCRIPTION
**Pull request by eldiegod**
fixes: #1089

### CODE REVIEW

File: src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ import {
   DeleteVideoContainer,
   FormWrapper,
   InputsContainer,
-  StyledHeaderTextField,
+  StyledTitleArea,
   StyledRadioContainer,
 } from './EditVideoForm.style'
```
Comment (drillprop) [inline src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx:48]: We have a warning here! ⚠️

Comment (eldiegod) [reply src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx:48]: fixed

File: src/shared/components/TitleArea/TitleArea.stories.tsx [hunk 2]
```diff
@@ -1,29 +1,29 @@
 import { Meta, Story } from '@storybook/react'
 import React, { useState } from 'react'
 
-import { HeaderTextField, HeaderTextFieldProps } from './HeaderTextField'
+import { TitleArea, TitleAreaProps } from './TitleArea'
 
 export default {
-  title: 'Shared/H/HeaderTextField',
-  component: HeaderTextField,
+  title: 'Shared/H/TitleArea',
```
Comment (kdembler) [inline src/shared/components/TitleArea/TitleArea.stories.tsx]:
```suggestion
  title: 'Shared/T/TitleArea',
```

File: src/shared/components/VideoTileBase/VideoTileBase.styles.tsx [hunk 3]
```diff
@@ -201,7 +201,7 @@ export const ProgressBar = styled.div`
   background-color: ${colors.blue['500']};
 `
 
-export const CoverVideoPublishingStateOverlay = styled.div`
+export const VideoHeroPublishingStateOverlay = styled.div`
```
Comment (kdembler) [inline src/shared/components/VideoTileBase/VideoTileBase.styles.tsx]: I don't think this is a desired change, this doesn't have anything to do with the `VideoHero` component

File: src/shared/components/ProgressDrawer/ProgressDrawer.stories.tsx [hunk 4]
```diff
@@ -1,15 +1,15 @@
 import { Meta, Story } from '@storybook/react'
 import React, { useState } from 'react'
 
-import { Checkout, CheckoutProps } from './Checkout'
+import { ProgressDrawer, ProgressDrawerProps } from './ProgressDrawer'
 
 export default {
-  title: 'Shared/C/Checkout',
-  component: Checkout,
+  title: 'Shared/C/ProgressDrawer',
```
Comment (kdembler) [inline src/shared/components/ProgressDrawer/ProgressDrawer.stories.tsx]:
```suggestion
  title: 'Shared/P/ProgressDrawer',
```

**Review by kdembler — CHANGES_REQUESTED** [review]

**Review by kdembler — APPROVED** [review]

**Pull request merged**